setup.py

```diff
@@ -105,17 +105,17 @@ def check_setuptools_features():
     url='https://github.com/BigchainDB/bigchaindb/',
     author='BigchainDB Contributors',
     author_email='dev@bigchaindb.com',
-    license='AGPLv3',
+    license='Apache Software License 2.0',
     zip_safe=False,
 
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'Topic :: Database',
         'Topic :: Database :: Database Engines/Servers',
         'Topic :: Software Development',
         'Natural Language :: English',
-        'License :: OSI Approved :: GNU Affero General Public License v3',
+        'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3.5',
```

Comment:
I think it's fine to remove `3.5`. We are actually using features from `3.6`.

Reply:
Okay I removed that line.